generators/chipyard/src/main/scala/ConfigFragments.scala

```diff
@@ -80,12 +79,6 @@ class WithRenumberHarts(rocketFirst: Boolean = false) extends Config((site, here
   case MaxHartIdBits => log2Up(up(BoomTilesKey, site).size + up(RocketTilesKey, site).size)
 })
 
-
-
-// ------------------
-// Multi-RoCC Support
```

Comment:
Was deleting this intentional?

Reply:
Yes. Currently, nothing else in that file has a large header. So to keep things consistent, I deleted it.